### PR TITLE
Updating the selenium-chrome-driver dependency.

### PR DESCRIPTION
**Pull request by Vipinofficial11**
Updating the Selenium-chrome-driver version to 4.1.3 to use netty version 4.1.75

### CODE REVIEW

**Review by itsankit-google — APPROVED** [review]
lgtm, please get an approval from @rmstar or @albertshau before merge.

**Review by albertshau — APPROVED** [review]

**Pull request merged**
